coff.ml

```diff
@@ -1047,7 +1047,7 @@ module Stacksize = struct
     let hdr_offset = int16 (read ic 0x3c 2) 0 in
```

Comment:
This is orthogonal to your patch, but isn't the `hdr_offset` 32-bit wide? https://en.wikipedia.org/wiki/Portable_Executable

Reply:
You're right, it's a `LONG`. I'll update the commit!

Reply:
If the offset fits in 16 bits, I suppose it's safe to forget the high-order bit from int32 to int.